src/_content/posts/volunteer-opportunities.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Volunteer at DjangoCon US: An Opportunity to Give Back and Grow"
+layout: post
+
+author: Monica Oyugi
+category: General
+date: 2024-07-27 05:00:00
+cover:
+    url: /assets/img/blog/registration-desk.jpg
+    alt: "Volunteers and attendees at registration desk"
```

Comment:
This causes the image to be displayed twice. I think its' intended to only appear above "Why Should You Volunteer"

Reply:
i think it might keep better flow in reading the post if we keep the cover and remove the one in the body text